top-sites/README.md

```diff
@@ -0,0 +1,9 @@
+# Top Sites
+
+## What it does
+
+This extension creates a new tab page that lists the topSites visited.
```

Comment:
It might be clearer to say something like: "this extension replaces the built-in page shown when the user opens a new tab without specifying a page to load (for example, when the user presses Ctrl+T). The replacement page is populated with links taken from the topSites API."